src/components/molecules/Common/AssetModal/AssetContainer/index.tsx

```diff
@@ -166,35 +209,34 @@ const AssetContainer: React.FC<Props> = ({
 };
 
 const Wrapper = styled.div`
-  height: 558px;
   width: 100%;
 `;
 
-const AssetWrapper = styled(Flex)`
-  height: 425px;
+const AssetWrapper = styled.div<{ isHeightFixed?: boolean }>`
+  height: ${({ isHeightFixed }) => (isHeightFixed ? "" : "425px")};
+  min-height: 400px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+`;
+
+const ButtonWrapper = styled.div<{ onRemove?: (assets: AssetType[]) => void }>`
+  display: flex;
+  justify-content: ${({ onRemove }) => (onRemove ? "flex-end" : "center")};
 `;
 
 const AssetList = styled(Flex)<{ layoutType?: LayoutTypes }>`
-  ${({ layoutType }) => layoutType === "list" && "flex-direction: column;"}
-  max-height: 458px;
+  ${({ layoutType }) => layoutType === "list" && "flex-direction: column;"};
+
```

Comment:
Be careful of that extra ";".  I am not sure it is fine in styled components, but that might be shown somewhere on the webpage :P As you can see, the conditional flex-direction has a ";" already because if layoutType is NOT a list, there is no flex-direction.
Also, becuase this is a Flex component you can (and should) put this up in the component itself (like `<Flex direction={layoutType === "list" ? "column" : undefined}>` ***Just double check that undefined works. Might need to be null***

Reply:
oh, yes, it's ture, thanks.